### DISCUSSION TITLE
Option to hide "Source" and other

### DISCUSSION THREAD

**Discussion by SmartFoxServer** [ankitskvmdam/clean-jsdoc-theme#301] [General]
Is there an option to hide the "Source" section? If not, any chance it can be added?
Also, it would be useful to be able to configure the "See" section containing multiple references as a list, instead of inline blocks (see attached image).
<img width="793" alt="Screenshot 2024-09-20 alle 10 16 43" src="https://github.com/user-attachments/assets/569c6190-dbfb-4ff6-9db9-6d01964dc3f2">
Other requirements may come later, as we just started to use clean-jsdoc-theme.
Thank you.

**Comment by ankitskvmdam** [comment]
In v5 you can hide the Source Files section with `outputSourceFiles: false` (and drop its sidebar link via `menu`). Docs: https://ankdev.me/clean-jsdoc-theme/theme/configuration/ Closing as resolved.
